docs/how-tos/create-custom-training-scripts.md

```diff
@@ -465,8 +467,9 @@ Use the `parse_filenames_and_labels_from_json` and `parse_filenames_and_bboxes_f
 
 Install any required dependencies and run your training script specifying the path to the <FILE>dataset.jsonl</FILE> file from your exported dataset:
 
-```sh {class="command-line" data-prompt="$" data-output="1-10"}
-python3 -m model.training --dataset_file=/path/to/dataset.jsonl --model_output_directory=.
+```sh {class="command-line" data-prompt="$"}
+python3 -m model.training --dataset_file=/path/to/dataset.jsonl \
+    --model_output_directory=. --args=custom_arg=3
```

Comment:
If we're doing this locally, it would just be 
```suggestion
    --model_output_directory=. --custom_arg=3
```

Reply:
good point - thanks!